src/test/fixtures/credential.mjs

```diff
@@ -0,0 +1,78 @@
+export const ropsten = {
+  network: "ropsten",
```

Comment:
if it's as someone new, it's not obvious these credentials are meant to be slice and dice into various configs later. might have the false impression these are the "correct" structures to follow for something somewhere else

Reply:
since we changed to typescript, do u think this issue is resolved ?